src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -911,6 +911,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     // - <none>
     void ControlCore::_refreshSizeUnderLock()
     {
+        if (_IsClosing())

```

Comment:
IsClosing only works on main thread. This is guaranteed to be on the main thread right?

Reply:
All callers appear to be WinUI related callbacks/functions.